server/src/server.js

```diff
@@ -451,6 +451,29 @@ app.post("/event", function(req, res) {
   });
 });
 
+app.post("/timing", function(req, res) {
+  let bodyObj = req.body;
+  if (typeof bodyObj !== "object") {
+    throw new Error(`Got unexpected req.body type: ${typeof bodyObj}`);
+  }
+  hashUserId(req.deviceId).then((userUuid) => {
+    let userAnalytics = ua(config.gaId, userUuid.toString(), {strictCidFormat: false});
+    if (config.debugGoogleAnalytics) {
+      userAnalytics = userAnalytics.debug();
+    }
+    let params = {
+      userTimingCategory: bodyObj.timingCategory,
+      userTimingVariableName: bodyObj.timingVar,
+      userTimingTime: bodyObj.timingValue,
+      userTimingLabel: bodyObj.timingLabel
+    };
+    userAnalytics.timing(params).send();
+    simpleResponse(res, "OK", 200);
```

Comment:
Previously `/timing` allowed batching.  That would make the code on the addon side a little more complicated though.  Perhaps keep the completed timing data in one object and the in progress ones in another, so whenever teh in progress list is empty and there are completed timing measurements, send those to `/timing`?

Reply:
Yeah, I noticed that too. Since `sendEvent` fires many more pings, and fires each one individually, it didn't seem worth bothering with the extra state management.

If there's a good reason to batch events in general, maybe we can adjust both endpoints in a followup bug?

Reply:
I made a followup in #3757